src/themes/dev/maps/color.scss

```diff
@@ -1,85 +1,2 @@
-$color: (
-  'blue': rgb(0, 68, 148),
```

Comment:
I think these can stay here as defaults as the eu can be considered an override I think

Reply:
additional comment: we'll have to merge the eu tokens in here

Reply:
I don't think we need to megre the EU tokens. The `with` option handles this well. If the item do not exists, it is created.

Reply:
it'll be very handy if there is no requirement for these, i hadn't tested it manually but really great if we can keep them separate and treat the eu as override

Reply:
It looks like it's not working as expected, unless i'm missing something all the missing keys in the map we use in the components scss don't generate any output in the css, this is the blockquote in eu-core, for instance, that uses `yellow`:
![image](https://user-images.githubusercontent.com/14326885/100356662-30af7000-2ffc-11eb-970d-5c6bff99fb8e.png)

Reply:
Yes, excellent catch. It's something we found and discuss indeed

Reply:
yes, just to be clear on one point: this PR just adds the EU tokens, but there are no relation between EC and EU colours (they don't even use the same token name).
So of course, they will be modification required at each component level (to use one of the colour available in EU), but that will be done when we have the corresponding specs. We can mitigate this by adding a few missing colours, but it won't solve everything

Reply:
As we discussed in the chat yesterday, it turns out that `@use` is not achieving what we aimed for in terms of final list of tokens. 

After spending a few hours trying to solve it, I can propose 2 options:

1) Use `map.merge` to actually merge the maps. The approach will have to change a bit and not be `with()` but go around modules and namespaces:

```scss
@use 'sass:map';

@use '@ecl/theme-dev/globals' with (
  $border-radius: 'yes'
);

@use '@ecl/theme-dev/maps/color' as *;

$color: map.merge(
  $color,
  (
    'blue-5': #f3f6fc,
    'blue-10': #e7edfa,
    'blue-20': #cfdaf5,
    'blue-40': #9fb5ea,
    'blue-60': #6e91e0,
    'blue-80': #3e6cd5,
    'blue-100': #0e47cb,
    'blue-110': #0d40b7,
    'blue-120': #0b39a2,
    'blue-130': #0a328e,
    'blue-140': #082b7a,
    'yellow-10': #fffae6,
    'yellow-20': #fff5cc,
    'yellow-40': #ffeb99,
    'yellow-60': #ffe066,
    'yellow-80': #ffd633,
    'yellow-100': #fc0,
    'yellow-110': #e6b800,
    'yellow-120': #cca300,
    'yellow-130': #b38f00,
    'yellow-140': #b38f00,
    'grey-5': #f4f5f5,
    'grey-10': #e9eaeb,
    'grey-20': #d4d5d7,
    'grey-40': #a8aaaf,
    'grey-60': #7d8088,
    'grey-80': #515560,
    'grey-100': #262b38,
    'grey-110': #222732,
    'grey-120': #1e222d,
    'grey-130': #1b1e27,
    'grey-140': #171a22,
    'purple-100': #510dcd,
  )
);
```

2) Place missing values in 

```scss
@use 'sass:map';

@use '@ecl/theme-dev/globals' with (
  $border-radius: 'yes'
);
@use '@ecl/theme-dev/maps/color' with (
  $color: (
    // defaults copy
    'blue': #004494,
    'yellow': #ffd617,
    'grey': #404040,
    'black': #000,
    'white': #fff,
    'blue-75': #4073af,
    'blue-50': #7fa1c9,
    'blue-25': #bfd0e4,
    'yellow-75': #ffde39,
    'yellow-25': #fff4bb,
    'grey-25': #cfcfcf,
    'grey-15': #e3e3e3,
    'blue-n': #006fb4,
    'orange': #f29527,
    'green': #467a39,
    'red-130': #981722,
    'red': #da2131,
    'primary': #004494,
    'secondary': #ffd617,
    'tertiary': #e3e3e3,
    'text': #404040,
    'white-100': #fff,
    'black-100': #000,
    'error': #da2131,
    'info': #006fb4,
    'success': #467a39,
    'warning': #f29527,
    'orange-100': #f29527,
    'green-100': #467a39,
    'red-100': #da2131,
    // new and overrides
    'blue-5': #f3f6fc,
    'blue-10': #e7edfa,
    'blue-20': #cfdaf5,
    'blue-40': #9fb5ea,
    'blue-60': #6e91e0,
    'blue-80': #3e6cd5,
    'blue-100': #0e47cb,
    'blue-110': #0d40b7,
    'blue-120': #0b39a2,
    'blue-130': #0a328e,
    'blue-140': #082b7a,
    'yellow-10': #fffae6,
    'yellow-20': #fff5cc,
    'yellow-40': #ffeb99,
    'yellow-60': #ffe066,
    'yellow-80': #ffd633,
    'yellow-100': #fc0,
    'yellow-110': #e6b800,
    'yellow-120': #cca300,
    'yellow-130': #b38f00,
    'yellow-140': #b38f00,
    'grey-5': #f4f5f5,
    'grey-10': #e9eaeb,
    'grey-20': #d4d5d7,
    'grey-40': #a8aaaf,
    'grey-60': #7d8088,
    'grey-80': #515560,
    'grey-100': #262b38,
    'grey-110': #222732,
    'grey-120': #1e222d,
    'grey-130': #1b1e27,
    'grey-140': #171a22,
    'purple-100': #510dcd,
  )
);
```

None of these are ideal in my opinion, though I tend to have preference for the second because it is ok to have all desired values in 1 place for easier access and reference and the first one is not a real solution but a work-around.

Let me know if you have other suggestions